pkg/cmd/taskrun/cancel_test.go

```diff
@@ -311,7 +322,7 @@ func TestTaskRunCancel_v1beta1(t *testing.T) {
 				Status: duckv1beta1.Status{
 					Conditions: duckv1beta1.Conditions{
 						{
-							Status: corev1.ConditionTrue,
+							Status: corev1.ConditionUnknown,
```

Comment:
This should be changed to Unknown since it needs to make it through the validation of whether or not the TaskRun has finished or not.